src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
  * Deprecate the `notifier.logger_notification_listener` service, use the `notifier.notification_logger_listener` service instead
  * Allow setting private services with the test container
  * Register alias for argument for workflow services with workflow name only
+ * Add `http_client.uri_template` configuration tree to enable using URI templates with `UriTemplateHttpClient`
```

Comment:
As discussed offline, I'd suggest always registering the decorator, so that the feature works out of the box, including the nice error message when "vars" is used but deps are missing